editor/plugins/tiles/tile_set_atlas_source_editor.cpp

```diff
@@ -789,9 +789,9 @@ void TileSetAtlasSourceEditor::_update_current_tile_data_editor() {
 	// Find the property to use.
 	String property;
 	if (tools_button_group->get_pressed_button() == tool_select_button && tile_inspector->is_visible() && !tile_inspector->get_selected_path().is_empty()) {
-		Vector<String> components = tile_inspector->get_selected_path().split("/");
-		if (components.size() >= 1) {
-			property = components[0];
+		const String &selected = tile_inspector->get_selected_path();
+		if (selected.get_slice_count("/") >= 1) {
```

Comment:
I think `get_slice_count()` always returns 1 or more, no? So the check is unnecessary.